src/spikegadgets_to_nwb/convert_position.py

```diff
@@ -15,6 +15,60 @@
 NANOSECONDS_PER_SECOND = 1e9
 
 
+def find_wrap_point(t):
```

Comment:
Can't you just use np.diff and find the negative numbers?

Reply:
Tried this first but  doesn't scale well with the size of the timestamps.  Binary search is faster for these big files